main_hall.py

```diff
@@ -0,0 +1,92 @@
+class Main_Hall:
+    floor1_is_flooded = True
+    if floor1_1_flooded = True:
+        def __int__(self,global_state,inventory):
+            self.global_state = global_state
+            self.inventory = inventory
+            self.set_state()
+            self.description = """ You are the end end of senior hall near the library.
+                               To the north there are stairs leading down to the main hall which looks flooded,
```

Comment:
I've normally seen these split up into separate sentences, but I don't know how we want to do the style. Discuss on Tuesday?